components/brave_extension/extension/brave_extension/manifest.json

```diff
@@ -64,6 +68,16 @@
       ],
       "run_at": "document_start",
       "all_frames": true
+    },
+    {
+      "matches": [
+        "*://*/*"
```

Comment:
can this be limited to http/https pages?

Reply:
nvm, that's the default behavior apparently https://developer.chrome.com/docs/extensions/mv3/match_patterns/